app/routes/install/index.js

```diff
@@ -23,6 +23,8 @@ const readFile = require('fs-readfile-promise');
 const axios = require('axios');
 const getBunyanConfig = require('../../utils/bunyan.js').getBunyanConfig;
 const { CLUSTER_REG_STATES } = require('../../apollo/models/const');
+const timeuuid = require('node-time-uuid');
```

Comment:
this package hasnt been published in 6 years and only has 150 weekly downloads. i think we probably want to stay away from this. Also node has a built in time library that i think will be sufficient for this use case. see next comment for a suggestion